docs/Design/Classes/Class Declarations.md

```diff
@@ -211,3 +226,4 @@ class Shape extends Measurements<Point> {
 * Interfaces do not exist in Python, so marking a class as implementing one is currently out of scope.
 * Adding modifiers such as `"extends"` to generics is currently out of scope.
 * Duck-typed languages such as Ruby and Python have no need for generics, so GLS will skip printing generic information in them.
+* Trying to implement interfaces in Python or Ruby will print nothing.
```

Comment:
As with the other review: it doesn't make sense to say classes won't exist in those language. It would be better to just leave out the "implements interface" part.